Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "libstratis"
-version = "2.0.1"
+version = "2.1.0"
 authors = ["Stratis Developers <stratis-devel@lists.fedorahosted.com>"]
 edition = "2018"
```

Comment:
Should the edition be updated? If so, Jenkins checks currently fail.

Reply:
There is currently no Rust 2020 edition. It's a little bit confusing but they only periodically release editions. The latest one that I'm aware of is 2018.

Reply:
There are rumours of a 2020 or 2021 edition, with better support for error management, but I haven't heard anything definite yet.